Makefile

```diff
@@ -127,8 +127,8 @@ down-x86:
 
 clear-standalone-data:
 	@docker run --rm -v "$(PWD)/$(STANDALONE_DIRECTORY)/data:/pwd" busybox \
-	sh -c "cd /pwd && rm -rf alertmanager/* clickhouse/* signoz/*"
+	sh -c "cd /pwd && rm -rf alertmanager/* clickhous*/* signoz/* zookeeper-*/*"
```

Comment:
@prashant-shahi any reason for removing `e` from `clickhouse`?

Reply:
yes, it is to include all of the following data folders:
- `clickhouse`
- `clickhouse-2`
- `clickhouse-2`

Reply:
though `clickhouse*/*` would also have done as  ending `e` is part of all